app/controllers/events_controller.rb

```diff
@@ -40,13 +38,18 @@ def destroy
     redirect_to root_path, status: :see_other
   end
 
+  def log_out
+    sign_out(current_user)
+    redirect_to root_path
+  end
+
```

Comment:
Why is this here??

Reply:
We don't quite understand, Should we move it to the Application controller?

Reply:
Devise already has a logout route

Reply:
You should use it instead of creating your own

Reply:
You should use it instead of creating your own

Reply:
Got it, thanks!